src/assets/i18n/en.json

```diff
@@ -168,6 +168,7 @@
     "OPEN_EXPLORER": "Open in Explorer",
     "QRCODE_COPY": "Tap on address to copy it to clipboard or click on icon to share QR code.",
     "RECEIVE": "Receive",
+    "RECEIVE_TOKEN": "Receive {{ Token }}",
```

Comment:
If you add this line you will need to add it to all other translations too.

Reply:
But I don't speak this languages? Any there are already like 7 translations missing anyway..so I thought I'd just omit it and the next guy who speaks the language adds it?

Reply:
For example in german I had to switch from `Erhalten {{ Token }} ` to `{{ Token }} Empfangen`, that's why I introduced the parameter.